ads/_config.js

```diff
@@ -57,6 +58,12 @@ export const adPreconnect = {
     'https://s.yieldmo.com',
     'https://ads.yieldmo.com',
   ],
+  revcontent: [
+    'https://trends.revcontent.com',
+    'https://cdn.revcontent.com',
+    'https://img.revcontent.com',
+    'https://labs-cdn.revcontent.com',
```

Comment:
So we just need the one entry above in **adPrefech**? Those host mentioned in pre-fetch is only used for the js service file.